src/goto-symex/goto_symex_state.h

```diff
@@ -267,9 +267,8 @@ class goto_symex_statet final
 
   // stack frames -- these are used for function calls and
   // for exceptions
-  class framet
+  struct framet
```

Comment:
In general people use class when there are virtual functions, but I don't think it's explicitly a CBMC convention.

Reply:
The C++ Core Guidelines has two guidelines which relate to this:
 * C.2: Use class if the class has an invariant; use struct if the data members can vary independently  
 * C.8: Use class rather than struct if any member is non-public